lib/src/view/analysis/tree_view.dart

```diff
@@ -151,23 +158,19 @@ class _InlineTreeViewState extends ConsumerState<AnalysisTreeView> {
       );
     }
 
-    return CustomScrollView(
-      slivers: [
+    return ListView(
```

Comment:
Why this change?

Reply:
To remove the blank space when there is no opening header
![Screenshot_20240916-170457.png](https://github.com/user-attachments/assets/f0c93c1a-92fd-4d68-ba7c-02d14b67ab79)

![Screenshot_20240916-170451.png](https://github.com/user-attachments/assets/b0d4bf68-e66b-42b7-b09f-017f363d1402)